_data/statuses.yaml

```diff
@@ -1,6 +1,7 @@
 - Last Call
 - Draft
 - Accepted
```

Comment:
If *Ready* is the new *Accepted*, why leave the old status here?

Reply:
I didn't update every affected EIP yet (and eip-validator needs an update too), so this was a temporary way to get this merged, then update tools and affected files, finally remove the old one.

Reply:
I'd argue though it would be better for me to split the renaming into a new PR so that we can merge the clarification parts.